README.md

```diff
@@ -53,7 +53,7 @@ cd `dirname $0`
 cd src/openlrw
 git pull
 mvn -DskipTests=true clean install
-cp target/openlrw-1.jar ../../lib/openlrw.jar
+cp target/matthews-1.jar ../../lib/openlrw.jar
```

Comment:
I remember @gryglbrt explained why the application is named "matthews" (See: ["Warehouse" by The Dave Matthews Band](https://youtu.be/uphTKT1-G_g)) and that there may be some effort underway to "correct" it to "openlrw".  @arwhyte and I thought it would be nice to keep the "matthews" name. However, I also feel the application name should be something obvious, so "openlrw" makes a lot of sense.

If the application name isn't changed, let's see about getting the build process to automatically name the output `target/openlrw-1.jar` instead.

Also, let's add a bit of text to this file that explains the "matthews" name.